modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinAbstractDmlStatementSelfTest.java

```diff
@@ -164,6 +165,19 @@ final CacheConfiguration binaryCacheConfig() {
         return cache;
     }
 
+    /** Clean tables to avoid DDL errors in repeated test runs.  */
+    protected void dropTables() throws SQLException {
+        List<List<?>> tablesLeft = execute(conn, "select CACHE_NAME, TABLE_NAME, SCHEMA_NAME from SYS.TABLES");
+        for( List<?> resultRecord : tablesLeft ){
+            String cacheName = resultRecord.get(0).toString();
+            if(cacheName.startsWith("SQL_")){
+                String tableName = resultRecord.get(1).toString();
+                String schemaName = resultRecord.get(2).toString();
+                execute(conn, "drop table "+schemaName+".\""+tableName+"\"");
```

Comment:
empty line required.

Reply:
Removed at all. Simplier approach involved: dropping table on the place: "drop table..."